test/distributed/org/apache/cassandra/fuzz/sai/AccordFullMultiNodeSAITest.java

```diff
@@ -18,11 +18,14 @@
 
 package org.apache.cassandra.fuzz.sai;
 
+import org.junit.Ignore;
+
 import org.apache.cassandra.harry.SchemaSpec;
 import org.apache.cassandra.harry.gen.Generator;
 import org.apache.cassandra.harry.gen.SchemaGenerators;
 import org.apache.cassandra.service.consensus.TransactionalMode;
 
+@Ignore("CASSANDRA-20567: Repair is failing due to missing SAI index files when using zero copy streaming")
```

Comment:
Is MultiNodeSaiTest also being ignored? It would be good to @Ignore that and reference this test so we know we need or remove the @Ignore here.

Reply:
nope, it doesn't fail.  There is something going on where these failing repairs cause corruption to accord

Reply:
for more context, `org.apache.cassandra.fuzz.sai.MultiNodeSAITestBase#repair`

```
    @Override
    protected void repair(SchemaSpec schema)
    {
        cluster.get(1).nodetool("repair", schema.keyspace);
    }
```

There is a big issue with this logic, it doesn't validate that the repair was successful!  So the repairs are failing, but that doesn't seem to impact the data for the non-accord case.

Yet for some reason accord produces incorrect results, so something about the repair modifies our behavior when it shouldn't.

I have not look into *why* this is happening, I just found that the stream is failing due to ZCS missing the Meta file.  Since this is incremental repair that should make it so the sstables never get promoted to repaired

Reply:
Posted to the ticket linked in the Ignore, the better way to do the repair is

```
cluster.get(1).nodetoolResult("repair", s2.metadata.keyspace, s2.metadata.name).asserts().success()
```

We also need a repair config change to avoid ephemeral issues in CI